app/features/notes_generator/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "inputs": [
+        {
+            "label": "Notes Generator Args.",
```

Comment:
Specify all the required args. in the `number` or `text` format.

Reply:
this is the new request sample:
```
{
   "user":{
      "id":"string",
      "fullName":"string",
      "email":"string"
   },
   "type":"chat",
   "tool_data":{
      "tool_id":16,
      "inputs": [
        {
            "name": "topic",
            "value": "Machine Learning"
        },
        {
            "name": "details",
            "value": "information about machine learning fundamentals and regression models"
        },
        {
            "name": "nb_columns",
            "value": 4
        },
        {
            "name": "orientation",
            "value": "portrait"
        },
        {
            "name": "file_urls",
            "value": "https://www.interactions.com/wp-content/uploads/2017/06/machine_learning_wp-5.pdf,https://firebasestorage.googleapis.com/v0/b/kai-ai-f63c8.appspot.com/o/uploads%2F510f946e-823f-42d7-b95d-d16925293946-Linear%20Regression%20Stat%20Yale.pdf?alt=media&token=caea86aa-c06b-4cde-9fd0-42962eb72ddd"
        },
        {
            "name": "file_types",
            "value": "pdf,pdf"
        },
        {
            "name": "langs",
            "value": "en,en"
        }
    ]
   }
}

```